apps/keyboard/js/layouts/uk.js

```diff
@@ -0,0 +1,85 @@
+Keyboards.uk = {
+  label: 'Ukrainian',
+  shortLabel: 'Uk',
+  menuLabel: 'українська',
+  imEngine: 'latin',
+  types: ['text', 'url', 'email', 'password'],
+  basicLayoutKey: 'АБВ',
+  lang: 'uk',
+  alt: {
+    а: '@',
```

Comment:
We have `@` defined in the 2nd page, for consistency with other layouts, would suggest don't provide it as an alternative char here.